azure-pipelines/vs-threading_OptProf.yml

```diff
@@ -0,0 +1,90 @@
+# OptProf profiling workflow for VS-Threading repository
```

Comment:
Can we drop this comment, which is redundant now that the file _is_ in the vs-threading repo?
Also can we rename the file to `OptProf_part2.yml`? We already have an `OptProf.yml` file, which serves as the kick-off that eventually triggers this one, so naming them in this way should help with understanding and discovery.